crates/dkg-cli/README.md

```diff
@@ -128,103 +140,93 @@ CONFIG_PATH=crates/key-server/key-server-config.yaml MASTER_SHARE_V0=0x208cd48a9
 
-export DKG_ENC_PK=$(jq -r '.enc_pk' dkg-state/dkg.key)
-export DKG_SIGNING_PK=$(jq -r '.signing_pk' dkg-state/dkg.key)
+#### Member Runbook
 
-# Register onchain. 
-sui client switch --env $NETWORK
-YOUR_SERVER_URL="replace your url here"
-MY_ADDRESS=$ADDRESS_0 # Replace your address here.
+1. Share with the coordinator your address (`MY_ADDRESS`). This is the wallet used for the rest of the onchain commands. Make sure you are on the right network with wallet with enough gas.
 
-sui client switch --address $MY_ADDRESS
-sui client call --package $COMMITTEE_PKG --module seal_committee \
-  --function register \
-  --args $COMMITTEE_ID x"$DKG_ENC_PK" x"$DKG_SIGNING_PK" "$YOUR_SERVER_URL"
-```
+2. Wait till the coordinator annouces Phase 1 and receive the `dkg-rotation.yaml` file containing `COMMITTEE_PKG`, `CURRENT_COMMITTEE_ID` and `COMMITTEE_ID` (next committee ID). Verify its parameters (members addresses, threshold, the current committee ID) on Sui Explorer. Add the following member specific fields to `dkg-rotation.yaml`:
 
-4. Wait for the coordinator to announce phase 2.
+```yaml
+MY_ADDRESS: 0x0636157e9d013585ff473b3b378499ac2f1d207ed07d70e2cd815711725bca9d
+MY_SERVER_URL: https://myserver.example.com
+```
 
-a. For continuing members, run the CLI below to initialize the local state and create your message file. Must provide `--old-share` arg with your current version `X` master share. Share the output file with the coordinator.
+And run the script to generate keys locally and register onchain.
 
 ```bash
-cargo run --bin dkg-cli create-message --my-address $MY_ADDRESS --committee-id $COMMITTEE_ID --network $NETWORK --old-share $MASTER_SHARE_VX
-
-# This creates a file: ./message_P.json (where P is your party ID).
+python crates/dkg-cli/scripts/dkg-scripts.py genkey-and-register -c crates/dkg-cli/scripts/dkg-rotation.yaml
 ```
 
-b. For new members, run the CLI below that initializes the local state. Do not provide old share.
+3. Wait for the coordinator to announce Phase 2 and run the following: 
+
+**For continuing members**: Required to pass your old master share as an argument. 
 
 ```bash
-cargo run --bin dkg-cli create-message --my-address $MY_ADDRESS --committee-id $COMMITTEE_ID --network $NETWORK
+python crates/dkg-cli/scripts/dkg-scripts.py create-message \
+    -c crates/dkg-cli/scripts/dkg-rotation.yaml \
+    --old-share <your MASTER_SHARE_VX from previous committee version>
 
-# No file is created or needed to be shared with the coordinator. 
+# This creates a file: message_P.json (where P is your party ID).
 ```
 
-5. Wait for the coordinator to announce phase 3 and share a directory `./dkg-messages` containing all messages. Process the directory locally.
+Share the output `message_P.json` file with the coordinator.
 
-```bash
-cargo run --bin dkg-cli process-all --messages-dir ./dkg-messages
+**For new members**: Just run the command without old share. 
```

Comment:
why is this needed?

Reply:
renamed in python for new members just do init-state which calls create-message under the hood with old-share=none.